uefi-services/src/lib.rs

```diff
@@ -95,6 +96,51 @@ pub fn init(st: &mut SystemTable<Boot>) -> Result {
     }
 }
 
+// Internal function for print macros
+#[doc(hidden)]
+pub fn _print(args: core::fmt::Arguments) {
+    unsafe {
+        if let Some(st) = &mut SYSTEM_TABLE {
+            st.stdout().write_fmt(args).expect("Failed to write to stdout");
+        } else {
+            panic!("SYSTEM_TABLE is None");
```

Comment:
I think it would be clearer if we used code similar to the above:

```rust
        let table_ref = SYSTEM_TABLE
            .as_ref()
            .expect("The system table handle is not available");
```

Reply:
👍